### PR TITLE
Add support for custom accesslog pattern

### LINKED ISSUE #78: Add support for custom accesslog pattern
See PR #77

### PR DESCRIPTION
**Pull request by HerveMARTIN**
I've add a parameter to be able to set a custom value for access log pattern.

Closes #78

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -261,6 +261,7 @@
   # host valves
   $singlesignon_valve         = false,
   $accesslog_valve            = true,
+  $accesslog_valve_pattern    = '%h %l %u %t &quot;%r&quot; %s %b',
```
Comment (antoineco) [inline manifests/init.pp:264]: This new parameter should be documented:

- in [manifests/config.pp](https://github.com/antoineco/aco-tomcat/blob/a5bad125302585b64e916c8dfadb41e87ce6a7d1/manifests/config.pp#L291)
- in [README.md](https://github.com/antoineco/aco-tomcat/blob/a5bad125302585b64e916c8dfadb41e87ce6a7d1/README.md#accesslog_valve)

It should also be added to [manifests/instance.pp](https://github.com/antoineco/aco-tomcat/blob/a5bad125302585b64e916c8dfadb41e87ce6a7d1/manifests/instance.pp#L237), which pretty much duplicates the rest of the class.

**Review by antoineco — CHANGES_REQUESTED** [review]
Thanks for the new feature! It just requires some documentation and duplication for the `instance` type, and then it will be ready for merging.

File: manifests/init.pp (hunk 1 above)
Comment (HerveMARTIN) [reply manifests/init.pp:264]: Done, is it enough?

Comment (antoineco) [reply manifests/init.pp:264]: Yup that's perfect, cheers

**Review by antoineco — APPROVED** [review]

**Pull request merged**